src/main/java/com/minecolonies/coremod/colony/buildings/AbstractSchematicProvider.java

```diff
@@ -366,7 +367,7 @@ public int getRotation()
     /**
      * Load updated TE data from the schematic if missing.
      */
-    protected void updateTEDataFromSchematic()
+    protected void saveUpdateTEDataFromSchematic()
```

Comment:
isnt it safe not save?^^

Reply:
the save one won't throw an exception, the other one will throw an exception